core/lib/spree/core/search/base.rb

```diff
@@ -46,8 +46,8 @@ def method_missing(name)
           end
         end
 
-        def respond_to_missing?(name)
-          @properties.key?(name) || super(name)
+        def respond_to_missing?(name, include_private)
```

Comment:
I think that second argument is optional, right?

```suggestion
        def respond_to_missing?(name, include_private = false)
```

Reply:
That's true for `respond_to?` but not for `respond_to_missing?` which should never be used directly and works just as an internal hook:

<img width="1033" alt="image" src="https://user-images.githubusercontent.com/1051/105489164-1f5caa00-5cb3-11eb-8dec-c8ca0327747f.png">